pkg/models/task_info.go

```diff
@@ -27,3 +30,146 @@ type TaskInfo struct {
 func (*TaskInfo) TableName() string {
 	return "tasks"
 }
+
+const Introspect = "introspect"
+
+type IntrospectPayload struct {
+	Url   string
+	Force bool
+}
+
+const Snapshot = "snapshot"
+
+type SnapshotPayload struct {
+	SnapshotIdent        *string
+	SyncTaskHref         *string
+	PublicationTaskHref  *string
+	DistributionTaskHref *string
+}
```

Comment:
yeah, agreed, although i wonder if that leads to a circular dependency?    If not, i'd prefer that.

Reply:
Yeah the payload structs were originally in the tasks package, but I used it in the models package to get the pulp data from a Snapshot task. That created a circular dependency between models and tasks, so I moved it to models.

Reply:
There shouldn't be a circular dependency because `pkg/tasks/payloads` would be a new package, for example you might have a file `pkg/tasks/payloads/repository_snapshot.go`

Reply:
Ohh got it